cfme/networks/balancer.py

```diff
@@ -0,0 +1,104 @@
+from utils import version
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+from cfme.web_ui import (
+    Region, Quadicon, toolbar as tb, paginator
+)
+from cfme.fixtures import pytest_selenium as sel
+from navmazing import NavigateToSibling, NavigateToAttribute
+from utils.appliance import Navigatable
+from utils.update import Updateable
+from cfme.common import Taggable, SummaryMixin
+from functools import partial
+
+
+pol_btn = partial(tb.select, 'Policy')
+details_page = Region(infoblock_type='detail')
+
+
+class Balancer(Taggable, Updateable, SummaryMixin, Navigatable):
+    ''' Class representing balancers in sdn '''
+    in_version = ('5.8', version.LATEST)
+    category = "networks"
+    page_name = 'network_balancer'
+    string_name = 'NetworkBalancer'
+    refresh_text = "Refresh items and relationships"
+    detail_page_suffix = 'network_balancer_detail'
+    quad_name = None
+    db_types = ["NetworkBalancer"]
+
+    def __init__(
+            self, name=None, key=None, appliance=None):
+        Navigatable.__init__(self, appliance=appliance)
+        self.name = name
+        self.key = key
+
+    def load_details(self):
```

Comment:
`load_details` and `get_detail` are convenience methods that existed in the framework before, but that we do not want to maintain or continue implementing moving forward. I'd say from load_details and get_detail here, any callers should directly use `navigate_to` and the view widgets to get `widgetastic_manageiq.SummaryTable` field values.